docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'OpenTelemetry'
+description: 'Use the OpenTelemetry output to send test results in OTEL metrics format.'
+weight: 00
+---
+
+# OpenTelemetry
+
+k6 can push test run metrics in a [OpenTelemetry metrics format](https://opentelemetry.io/docs/specs/otel/metrics/) to a OTEL metrics collector or a metrics backend that support OTEL metrics fromat by using the [xk6-output-opentelemetry extension](https://github.com/grafana/xk6-output-opentelemetry/).
+
+This is an extension, and we are looking for feedback and contributions and aiming to graduate it as an experimental output within k6 core in one of the future releases. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
```

Comment:
```suggestion
{{< admonition type="note" >}}

This is an extension, and we're looking for feedback and contributions and aiming to graduate it as an experimental output within Grafana k6 in a future release. Feel free to [open an issue](https://github.com/grafana/xk6-output-opentelemetry/issues) if you have any feedback or suggestions.

{{< /admonition >}}
```

Just a suggestion, I think we can switch this to an admonition and move it up to line 8 (between the H1 and the first paragraph) to make it more similar to how we use the caution admonition for experimental modules.